tests/manual-test-cases/Group5-Functional-Tests/5-8-DRS.robot

```diff
@@ -79,14 +79,13 @@ DRS Setup
 *** Test Cases ***
 Test
     Log To Console  \nStarting test...
-    ${status}  ${message}=  Run Keyword And Ignore Error  Install VIC Appliance To Test Server  certs=${false}  vol=default
-    Should Contain  ${message}  DRS must be enabled to use VIC
-    Should Be Equal As Strings  ${status}  FAIL
+    Install VIC Appliance To Test Server  certs=${false}  vol=default
+    Run Regression Tests
+    Cleanup VIC Appliance On Test Server
 
     Log To Console  Enable DRS on the cluster
```

Comment:
@mhagen-vmware Thanks. Yes, the log is for enabling DRS which I did not notice before.